test/README.md

```diff
@@ -1,18 +1,97 @@
 # Running Tests
 
-To run the tests, perform the following steps:
+To run all tests, perform the following steps:
 
-1) Startup a fresh iRODS server.
-2) Startup the Testing keycloak image.
-3) Startup the iRODS HTTP API server.
-4) Ensure the HTTP API can communicate with the iRODS server and the keycloak image.
-5) Configure the [config.py](config.py) to the appropriate test configuration.
-6) Install `pytest` for the OpenID tests.
-7) Run `pytest` to run all tests. If desired, run `test_irods_http_api.py` directly if OpenID tests are not required.
+## Startup a fresh iRODS server
 
-## Testing support for OpenID Connect via Keycloak
+Use the iRODS testing environment to setup an iRODS server: https://github.com/irods/irods_testing_environment
+
+### Get the iRODS network information
+
+We need the network that the HTTP API is running on to have all pieces of the server
+communicate with each other.
+
+Following is an example of how to list all available docker networks with examples
+of how the output may appear:
+
+```console
+$ docker network ls
+NETWORK ID     NAME                              DRIVER    SCOPE
+d06849108e9c   bridge                            bridge    local
+371117285a13   host                              host      local
+fbf586b6459f   none                              null      local
+297992bf57a9   ubuntu-2204-postgres-14_default   bridge    local
+```
+
+In the example output, the desired network would be `ubuntu-2204-postgres-14_default`.
```

Comment:
Maybe it would help to make it explicitly clear that this network was created by the testing environment and the container running iRODS is connected to it.

Reply:
Done.